tests/Supportive/OutputFormatter/MermaidJSOutputFormatterTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tests\Qossmic\Deptrac\Supportive\OutputFormatter;
+
+use PHPUnit\Framework\TestCase;
+use Qossmic\Deptrac\Contract\Analyser\AnalysisResult;
+use Qossmic\Deptrac\Contract\Ast\DependencyType;
+use Qossmic\Deptrac\Contract\Ast\FileOccurrence;
+use Qossmic\Deptrac\Contract\OutputFormatter\OutputFormatterInput;
+use Qossmic\Deptrac\Contract\Result\Allowed;
+use Qossmic\Deptrac\Contract\Result\OutputResult;
+use Qossmic\Deptrac\Contract\Result\Violation;
+use Qossmic\Deptrac\Core\Ast\AstMap\ClassLike\ClassLikeToken;
+use Qossmic\Deptrac\Core\Dependency\Dependency;
+use Qossmic\Deptrac\Supportive\Console\Symfony\Style;
+use Qossmic\Deptrac\Supportive\Console\Symfony\SymfonyOutput;
+use Qossmic\Deptrac\Supportive\OutputFormatter\Configuration\FormatterConfiguration;
+use Qossmic\Deptrac\Supportive\OutputFormatter\MermaidJSOutputFormatter;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\BufferedOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Tests\Qossmic\Deptrac\Supportive\OutputFormatter\data\DummyViolationCreatingRule;
+
+class MermaidJSOutputFormatterTest extends TestCase
+{
+    /**
+     * @dataProvider dataForTestFinish
```

Comment:
A dataProvider with just one set is a bit over engineered, just inline the expected format.